packages/playground/app/_sass/vendor/normalize.scss/package.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
If we use a normalize why we need their package.json file?

Reply:
Done